redisbus/redisbus.go

```diff
@@ -340,18 +281,14 @@ func (r *RedisBus[M]) consumeMessages(ctx context.Context) error {
 
 		for {
 			select {
-			case <-ctx.Done():
+			case <-psc.Done():
 				// context signaled to stop listening
 				return
 			default:
 			}
 
-			msg, err := r.psc.Receive(ctx)
+			msg, err := psc.Receive()
 			if err != nil {
-				// TODO: timeout review..?
```

Comment:
We're using `Receive()` which is blocking and won't return a timeout, we can remove this bit.